DANS-KNAW/easy-split-multi-deposit
# Easy 1124 - DCT_TEMPORAL moet een SCHEME meekrijgen

fixes EASY-1124

#### When applied it will
* [x] do some arbitrary refactorings that I had done a while ago but never committed
* [x] add `DCT_TEMPORAL_SCHEME` handling when creating DDM from the CSV
* [x] add `DC_SUBJECT_SCHEME handling` when creating DDM from the CSV
* [x] modify documentation accordingly
* [x] remove the unused `appHomeDir` parameter from the settings
* [x] remove the old `DC_CREATOR` and `DC_CONTRIBUTOR` fields from the code (not now, but the TODOs that were added earlier in this PR have been removed again)

#### Where should the reviewer @DANS-KNAW/easy start?
`AddDatasetMetadataToDeposit`

#### How should this be manually tested?
* take a 'multi-deposit' (see test resources) and run the application on it. The resulting `bag/metadata/dataset.xml` should contain the proper `xsi-type` in the `temporal` tag. An example of this is
```xml
<dcterms:temporal xsi:type="abr:ABRperiode">PALEOLB</dcterms:temporal>
```
* transform the resulting bag from the previous step into an SDO-set using easy-stage-dataset. The `dataset/EMD` xml-file should contain the proper `coverage` tag. (Note that for this step you require https://github.com/DANS-KNAW/easy-app/pull/36 as well!) An example of this is
```xml
<emd:coverage>
        <dct:temporal eas:scheme="ABR" eas:schemeId="archaeology.dcterms.temporal">PALEOLB</dct:temporal>
```

* a similar transformation should happen for the DC_SUBJECT_SCHEME. The DDM should look like
```xml
<ddm:dcmiMetadata>
        <dc:subject xsi:type="abr:ABRcomplex">IX</dc:subject>
```
* the EMD should look like
```xml
<emd:subject>
        <dc:subject eas:scheme="ABR" eas:schemeId="archaeology.dc.subject">IX</dc:subject>
```

#### related pull requests on github
repo                       | PR
------------------ | -----------------
easy-app               | https://github.com/DANS-KNAW/easy-app/pull/36
